resources/views/posts/show.blade.php

```diff
@@ -0,0 +1,23 @@
+@extends('layout')
+@section('title', $post->title)
+@section('content')
+    <a class="btn btn-primary" href="{{url()->previous()}}">Back</a>
+    <table class="table table-striped">
+        <thead>
+        <th>ID</th>
+        <th>Title</th>
+        <th>Body</th>
+        <th>Created At</th>
+        <th>Updated At</th>
+        </thead>
+        <tbody>
```

Comment:
could you make it vertical table instead of horizontal. It would be really hard to read the contents. It would have you make two `tr`s of same length and no `thead` . one `tr` would have the `th`s and the otherone values.

Reply:
Without adding some CSS, your suggestion just creates a _different_ horizontal table (unless I misinterpreted something).